src/mlpack/methods/ann/loss_functions/dice_loss.hpp

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file dice_loss.hpp
+ * @author N Rajiv Vaidyanathan
+ *
+ * Definition of the dice loss function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LOSS_FUNCTIONS_DICE_LOSS_HPP
+#define MLPACK_METHODS_ANN_LOSS_FUNCTIONS_DICE_LOSS_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The dice loss performance function measures the network's
+ * performance according to the dice coefficient
+ * between the input and target distributions.
+ *
+ * For more information see the following.
+ *
+ * @inproceedings{milletari2016v,
+ *   title = {V-net: Fully convolutional neural networks
+ *            for volumetric medical image segmentation},
+ *   author = {Milletari, Fausto and Navab, Nassir and Ahmadi, Seyed-Ahmad},
```

Comment:
Please use the same format as given on the `arxiv` [page](https://arxiv.org/abs/1606.04797)

Reply:
@ShikharJ I had used the format given in [Google Scholar](https://scholar.googleusercontent.com/scholar.bib?q=info:3hH5LL0SWYsJ:scholar.google.com/&output=citation&scisig=AAGBfm0AAAAAXFFNOY1z9qdSgLlNzWMXUwX0utxRcDfO&scisf=4&ct=citation&cd=-1&hl=en)

Reply:
I would go with the ``IEEE`` publication as well, but up to you.

Reply:
I have updated this with `BibTeX` version from `arxiv`. Please let me know if it is fine.